<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>410</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>910</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sorts Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="b_calc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_save">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Save in file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_load">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_show">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Show result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_exit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_is">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert sort</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_bs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buble sort</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_qs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick sort</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_ss">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Selection sort</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_hs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Heap sort</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_sorts">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sorts:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_results">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Text results:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tb_res">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>530</width>
      <height>355</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>530</width>
      <height>355</height>
     </size>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>11</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Insert sort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Buble sort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quick sort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Selection sort</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Heap sort</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEnabled</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="b_plots">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Plots</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
